_opensearch/query-dsl/geo-bounding-box.md

```diff
@@ -0,0 +1,226 @@
+## Example
+
+You can use a geo-bounding box query to search for documents that contain geopoints. 
+
+Create a mapping with the `point` field mapped as `geo_point`:
+
+```json
+PUT testindex1
+{
+  "mappings": {
+    "properties": {
+      "point": {
+        "type": "geo_point"
+      }
+    }
+  }
+}
+```
+
+Index three geopoints as objects with latitudes and longitudes:
+
+```json
+PUT testindex1/_doc/1
+{
+  "point": { 
+    "lat": 74.00,
+    "lon": 40.71
+  }
+}
+
+PUT testindex1/_doc/2
+{
+  "point": { 
+    "lat": 72.64,
+    "lon": 22.62
+  } 
+}
+
+PUT testindex1/_doc/3
+{
+  "point": { 
+    "lat": 75.00,
+    "lon": 28.00
+  }
+}
+```
+
+Search for all documents and filter those documents whose points lie within the rectangle defined in the query:
+
+```json
+GET testindex1/_search
+{
+  "query": {
+    "bool": {
+      "must": {
+        "match_all": {}
+      },
+      "filter": {
+        "geo_bounding_box": {
+          "point": {
+            "top_left": {
+              "lat": 75,
+              "lon": 28
+            },
+            "bottom_right": {
+              "lat": 73,
+              "lon": 41
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+The response contains the matching document:
+
+```json
+{
+  "took" : 20,
+  "timed_out" : false,
+  "_shards" : {
+    "total" : 1,
+    "successful" : 1,
+    "skipped" : 0,
+    "failed" : 0
+  },
+  "hits" : {
+    "total" : {
+      "value" : 1,
+      "relation" : "eq"
+    },
+    "max_score" : 1.0,
+    "hits" : [
+      {
+        "_index" : "testindex1",
+        "_id" : "1",
+        "_score" : 1.0,
+        "_source" : {
+          "point" : {
+            "lat" : 74.0,
+            "lon" : 40.71
+          }
+        }
+      }
+    ]
+  }
+}
+```
+
+The preceding response does not include the document with a geopoint of `"lat": 75.00, "lon": 28.00` because of the geopoint's limited [precision](#precision).
+{: .note}
+
+## Precision
+
+Geopoint coordinates are always rounded down at index time. At query time, the bounding box upper boundaries are rounded down, and the lower boundaries are rounded up. Therefore, the documents with geopoints that lie on the lower and left edges of the bounding box might not be included in the results due to rounding error. On the other hand, geopoints that lie on the upper and right edges of the bounding box might be included in the results even though they are outside the boundaries. The rounding error is less than 4.20 &times; 10<sup>&minus;8</sup> degrees for latitude and less than 8.39 &times; 10<sup>&minus;8</sup> degrees for longitude (around 1 cm). 
+
+## Specifying the bounding box
+
+You can specify the bounding box by providing any of the following combinations of its vertex coordinates:
+
+- `top_left` and `bottom_right`
+- `top_right` and `bottom_left`
+- `top`, `left`, `bottom`, and `right`
+
+The following example specifies the bounding box using the `top`, `left`, `bottom`, and `right` coordinates:
```

Comment:
The example doesn't itself specify the bounding box. "The following example shows how to specify the bounding box"?